### PR TITLE
Main fd2 tabs test

### PR DESCRIPTION
**Pull request by jamesng5**
__Pull Request Description__

Address #199 

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

**Comment by johnmaccormick** [comment]
@jamesng5 @vuphuongha @tainguyen103 -- I haven't looked at your changes yet. I will examine those and comment separately about it. Meanwhile, there is something you need to fix with your PR. This PR should have only one changed file -- fd2tabsexistence.spec.js

The other changed files are part of a different change for a different issue, right? Therefore, you need to base your feature branch for this PR off the main branch, not a previous feature branch.

Please try to fix this on your own but if you need a hint of how to manipulate the branches and files, let me know and I will provide assistance.

File: farmdata2/farmdata2_modules/cypress/fd2tabsexistence.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    /**
+     * Check the non-existence of FieldKit, BarnKit and FD2 Config 
+     * tabs if a guest account is logged in.
+     */
+    context('Guest account is logged in', () => {
+    
+        beforeEach(()=>{
+            cy.login("guest", "farmdata2")
+            cy.visit('/farm')
```
Comment (johnmaccormick) [inline farmdata2/farmdata2_modules/cypress/fd2tabsexistence.spec.js:61]: If we change `guest` to `manager1` here, the tests should fail, right? However, when I did that, the tests passed. Therefore, the tests are not actually checking what you think they are checking. See if you can figure out how to fix this problem. If you need an additional hint, please ask. Thanks!

PS. same problem with previous set of tests.

**Comment by johnmaccormick** [comment]
Another comment: the file name for your test is hard to read. Please use camel case to make it more readable. Let me know if you need additional explanation of this. Good luck making these additional changes and getting your PR merged -- excellent progress so far!

File: farmdata2/farmdata2_modules/cypress/fd2tabsexistence.spec.js (hunk 1 above)
Comment (jamesng5) [reply farmdata2/farmdata2_modules/cypress/fd2tabsexistence.spec.js]: I fixed this. Can you please take a look at it? Thank you.

**Comment by johnmaccormick** [comment]
nice work! -- everything looks great -- merging now.

**Pull request merged**